compute/cloud-client/src/test/java/compute/disks/HyperdisksIT.java

```diff
@@ -96,15 +96,15 @@ public void stage1_CreateHyperdiskTest()
     Assert.assertTrue(hyperdisk.getZone().contains(ZONE));
   }
 
-  @Disabled
   @Test
   public void stage1_CreateHyperdiskStoragePoolTest()
       throws IOException, ExecutionException, InterruptedException, TimeoutException {
     String poolType = String.format("projects/%s/zones/%s/storagePoolTypes/hyperdisk-balanced",
         PROJECT_ID, ZONE);
     StoragePool storagePool = CreateHyperdiskStoragePool
          .createHyperdiskStoragePool(PROJECT_ID, ZONE, STORAGE_POOL_NAME, poolType,
-         "advanced", 10240, 10000, 10240);
+         "advanced", 10240, 10000, 10240,
```

Comment:
should the capacity to be `1024` and not `10240`?

Reply:
I do not understand what you mean by "creating a new task"? Do you propose to address code review comments made in this PR by submitting a separate PR?
If you do, I find this proposal very inconvenient and time consuming. Please, explain what do you mean.

Regarding my question about the value `10240`, I looked into Cloud Console UI and it limits the range to be from 1 to 1024 TB. I do not know if client library limits the range but I think that making it matching UX in Console is preferred.

Reply:
Fixed